check/README.md

```diff
@@ -0,0 +1,114 @@
+# Tool for work with string tables
+
+## Glossary
+
+- `string table` a file with translation
+- `reference table` en.txt string table.
+  English is the development language, so any changes related to text should be in that file.
+  Also, this table us used as fallback for other translations, if a needed key is missing.
+  This file is mostly maintained by people who change game code,
+  so the list of entries in this file should be the most up-to-date.
+- `entry` one piece of translation, it consists of
+  - `key` a sting that represents entry in the codebase (C++, Python).
```

Comment:
stringtable keys aren't just stuff from C++ and Python. They can also be used in FOCS and can be added just for substitution purposes via `[[KEY_INSERT_MACROS]]` within other strings in the same stringtable, and should also look in the reference (en) stringtable if not found in the current stringtable.